
Wed, 08 Oct 2008 02:20:01 GMT

MAKE:
mv pidigits.python-4.python pidigits.python-4.py
/usr/local/bin/python -OO -c "from py_compile import compile; compile('pidigits.python-4.py')"

COMMAND LINE:
/usr/local/bin/python pidigits.python-4.pyo 2000

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "pidigits.python-4.py", line 7, in <module>
    import gmpy
ImportError: No module named gmpy
